alibi_detect/saving/tests/test_saving.py

```diff
@@ -350,7 +353,7 @@ def test_save_ksdrift_nlp(data, preprocess_fn, max_len, enc_dim, tmp_path):
                  p_val=P_VAL,
                  preprocess_fn=preprocess_fn,
                  preprocess_at_init=True,
-                 input_shape=(max_len,),
+                 input_shape=(768,),  # hardcoded to bert-base-cased for now
```

Comment:
Why/what was the bug for using `max_len` parameter previously?

Reply:
The `input_shape` should be the model input data's shape, after it has passed through any nlp embedding and tokenizer (i.e. 768 for bert-base-cased). I had incorrectly set it to `max_len`, which is passed to the HuggingFace tokenizer's `max_length` kwarg, which sets the max length for the tokenizer to pad/truncate sentences to. The bug doesn't actually cause a problem at the moment, but it will for subclassed tensorflow models in the future (once we moved to `SavedModel` format).